docs/topics/proxy-servers.md

```diff
@@ -0,0 +1,150 @@
+# AKS Engine and proxy servers
+
+Using AKS engine on [Azure Stack](azure-stack.md) in connected and disconnected environments often times requires the use of a non-transparent proxy server. Non-transparent means that they're not part of the default route and have to be configured to be used.

```

Comment:
nit: use "AKS Engine" everywhere (instead of lowercase "AKS engine")

Reply:
done, replace AKS engine with AKS Engine (with an uppercase E)